bitnami/opensearch/templates/snapshots/init-job.yaml

```diff
@@ -3,62 +3,142 @@ Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if $.Values.snapshots.enabled -}}
-{{- $host := (include "opensearch.service.name" $) -}}
-{{- $port := (include "opensearch.service.ports.restAPI" $) -}}
+{{- if .Values.snapshots.enabled }}
 apiVersion: batch/v1
 kind: Job
 metadata:
-  annotations:
-    helm.sh/hook: post-install
-  {{- if .Values.commonAnnotations }}
-  {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
-  {{- end }}
-  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
-    app.kubernetes.io/component: snapshots
   name: {{ include "opensearch.snapshots.fullname" . }}
   namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" . ) | nindent 4 }}
+    app.kubernetes.io/component: snapshots
+  {{- $defaultAnnotations := dict "helm.sh/hook" "post-install" "helm.sh/hook-delete-policy" "hook-succeeded" }}
```

Comment:
Now that we support Kubernetes 1.23+ Have you considered using the field [ttlSecondsAfterFinished](https://kubernetes.io/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically)?

Reply:
To be done on follow-up